src/mainwindow.cpp

```diff
@@ -378,6 +378,37 @@ void PointViewerMainWindow::handleMessage(QByteArray message)
     {
         m_geometries->clear();
     }
+    else if (commandTokens[0] == "UNLOAD_FILES")
+    {
+        QString regex_str = commandTokens[1];
+        QRegExp regex(regex_str, Qt::CaseSensitive, QRegExp::WildcardUnix);
+        if (!regex.isValid())
+        {
+            tfm::format(std::cerr, "Invalid regular expression '%s': %s\n", 
+                        regex_str, regex.errorString()); 
+            return;
+        }
+        m_geometries->unloadFiles(regex);
+    }
+    else if (commandTokens[0] == "SET_VIEW_POSITION")
+    {
+        if (commandTokens.size()-1 != 3)
+        {
+            tfm::format(std::cerr, "Expected three coordinates, got %d\n",
+                        commandTokens.size()-1);
```

Comment:
I think this is fine going to cerr btw (in contrast to the regex above) - if this error ever happens it's most probably an internal bug in displaz rather than user error.